pubsub/kafka/kafka_test.go

```diff
@@ -40,16 +40,49 @@ func getBaseMetadata() pubsub.Metadata {
 	return m
 }
 
-func TestParseMetadata(t *testing.T) {
+func getCompleteMetadata() pubsub.Metadata {
 	m := pubsub.Metadata{}
 	m.Properties = map[string]string{
 		"consumerGroup": "a", "clientID": "a", "brokers": "a", "authRequired": "false", "maxMessageBytes": "2048",
 		skipVerify: "true", clientCert: clientCertPemMock, clientKey: clientKeyMock, caCert: caCertMock,
 		"consumeRetryInterval": "200",
 	}
+	return m
+}
+
+func TestParseMetadata(t *testing.T) {
 	k := getKafkaPubsub()
-	meta, err := k.getKafkaMetadata(m)
-	assert.Nil(t, err)
+	t.Run("default kafka version", func(t *testing.T) {
+		m := getCompleteMetadata()
+		meta, err := k.getKafkaMetadata(m)
+		require.NoError(t, err)
+		assert.NotNil(t, meta)
+		assertMetadata(t, meta)
```

Comment:
Nit: `assertMetadata` could be called once outside of the nested tests

Reply:
there are 2 independent tests where need to `assert` each own `metadata`. Maybe I'm missing the ask here.

Reply:
I think I misread the code. Disregard :)